lib/migration.js

```diff
@@ -412,92 +397,162 @@ function mixinMigration(MySQL, mysql) {
         }
       }
     });
+    if (sql.length > 0) {
+      self.applySqlChanges(model, sql, function(err, result) {
```

Comment:
Compared of old method `getAlterStatement`, the refactored `applySqlChanges` doesn't do a "join" of sql, while actually here `sql` is `sqls`, which is an array of `sql`.

I will double check is `applySqlChanges` aimed to apply a single sql statement or join and apply multiple ones.

Old method `getAlterStatement`:
```js
MySQL.prototype.getAlterStatement = function(model, statements) {
    return statements.length ?
      'ALTER TABLE ' + this.tableEscaped(model) + ' ' + statements.join(',\n') :
      '';
  };
```

New `applySqlChanges`
```js
MySQL.prototype.applySqlChanges = function(model, pendingChanges, cb) {
    var self = this;
    var thisQuery = 'ALTER TABLE ' + self.tableEscaped(model) + ' ' + pendingChanges.toString();
    thisQuery = thisQuery + ';';
    if (pendingChanges.length) {
      self.execute(thisQuery, cb);
    }
  };
```